bgpd/bgp_vty.c

```diff
@@ -4007,7 +4007,7 @@ DEFUN_YANG(bgp_listen_limit,
 
 DEFUN_YANG(no_bgp_listen_limit,
 	   no_bgp_listen_limit_cmd,
-	   "no bgp listen limit [(1-5000)]",
+	   "no bgp listen limit [(1-50000)]",
```

Comment:
is it meaningful to offer a number greater than the number of fds available? see libfrr.c, and thread.c

Reply:
:man_shrugging:

Reply:
thanks for feedback. I think the limit can also be set in systemd unit file.
but what if we set it to infinity, and if we dont pass parameter --limit-fd to daemon, what would be the problem to increase it?

Reply:
I think  that the problem that will occur will not be a file descriptor resource problem, but a cpu issue in handling all those fds all at once.

Reply:
well, there are fixed-size fd arrays to do things like 'poll', so if you can't fit the fds...

frr uses the command-line limit, or the OS's value: again, what would the point be of offering to configure more fds than can be used?

Reply:
A 5k listen limit was/is already past the normal 1k limit that we have for FD's.  I don't see a whole bunch of sense worrying about it.  Let's just note it in the documentation and how to extend the available FD's and move on.

Reply:
added some comment to explain the bgp issue.